docs(traceableai) - Initial docs for traceable.ai plugin (#7020)
Kong/docs.konghq.com: app/_hub/traceableai/traceableai/overview/_index.md

---
nav_title: Overview
title: Overview
---

> Traceable's Kong plugin lets traceable capture a copy of the API traffic that is flowing through Kong's API Gateway. Using this data Traceable is able to create a security posture profile of  APIs hosted on Kong. Traceable plugin is also capable of blocking traffic coming from malicious actors and IPs into Kong API Gateway.

Reviewer: ```suggestion
Traceable's Kong plugin lets Traceable capture a copy of the API traffic that is flowing through {{site.base_gateway}}. 
Using this data, Traceable is able to create a security posture profile of APIs hosted on Kong. 
Based on its findings, the Traceable plugin can also block traffic coming from malicious actors and IPs into {{site.base_gateway}}.
```
Some rephrasing to make the description flow together. Using the `{{site.base_gateway}}` variable, which will resolve to "Kong Gateway" in the output.

## How it works

> The Traceable Kong Plugin captures request and response data to forward to a locally running Traceable Module Extension(TME).

Reviewer: ```suggestion
The Traceable Kong Plugin captures request and response data to forward to a locally running Traceable Module Extension (TME).
```

## Prerequisites
The Traceable Kong Plugin requires a Traceable Platform Agent to be deployed in your environment. 

> For complete deployment instructions of the Traceable Platform Agent please visit the [traceable.ai docs site](https://docs.traceable.ai/docs/k8s).

Reviewer: ```suggestion
For complete deployment instructions of the Traceable Platform Agent, visit the [traceable.ai docs site](https://docs.traceable.ai/docs/k8s).
```

## How to install

> Once you have deployed a Traceable Platform Agent you are ready to install the plugin using the LuaRocks package manager:

Reviewer: ```suggestion
Once you have deployed a Traceable Platform Agent, you are ready to install the plugin using the LuaRocks package manager:
```

1. Install the traceable plugin:

    ```sh
    luarocks install kong-plugin-traceable
    ```

2. Update your loaded plugins list in {{site.base_gateway}}.

    In your `kong.conf`, append `traceable` to the `plugins` field. Make sure the field is not commented out.

    ```yaml
    plugins = bundled,traceable
    ```

3. Restart {{site.base_gateway}}:

    ```sh
    kong restart
    ```

## Using the plugin

> Register the Traceable plugin:

Reviewer: ```suggestion
Enable the Traceable plugin on a service:
```

```shell

> $ curl -X POST http://<kong-domain>:<kong-port>/services/<your-kong-service-id>/plugins/ \

Reviewer: ```suggestion
curl -X POST http://localhost:8001/services/{serviceId}/plugins/ \
```
We use `localhost:8001` as the default value in all examples. You can add a sentence to tell the readers to change the values to their own domain and port, if you like.
